### PR TITLE
feat: add truncate options

### PR DESCRIPTION
**Pull request by peterdeme**
Part of CHAT-2622

Note: this repo should require at least 1 reviewer. I can merge it myself which is not good.

### CODE REVIEW

File: lib/GetStream/StreamChat/Channel.php [hunk 1]
```diff
@@ -266,13 +266,12 @@ public function delete()
     }
 
     /**
+      * @param array $options
       * @return mixed
       * @throws StreamException
       */
-    public function truncate()
+    public function truncate($options=null)
```
[the remark this answers is not in the document]
Comment (peterdeme) [reply lib/GetStream/StreamChat/Channel.php:273]: Here's one: https://github.com/GetStream/stream-chat-php/blob/main/tests/integration/IntegrationTest.php#L639

[the remark this answers is not in the document]
Comment (AnatolyRugalev) [reply lib/GetStream/StreamChat/Channel.php:273]: It doesn't test this method with `null` options though

[the remark this answers is not in the document]
Comment (peterdeme) [reply lib/GetStream/StreamChat/Channel.php:273]: refactored it a little bit @AnatolyRugalev

[the remark this answers is not in the document]
Comment (AnatolyRugalev) [reply lib/GetStream/StreamChat/Channel.php:273]: This looks better, but test is still missing. I don't mind merging this though

**Review by AnatolyRugalev — APPROVED** [review]

File: lib/GetStream/StreamChat/Channel.php (hunk 1 above)
[the remark this answers is not in the document]
Comment (peterdeme) [reply lib/GetStream/StreamChat/Channel.php:273]: Okay, I can add one for `truncate(null)`

**Pull request merged**